packages/nextjs/src/server/utils.ts

```diff
@@ -242,10 +257,12 @@ export function encryptClerkRequestData(requestData?: Partial<AuthenticateReques
     return;
   }
 
-  return AES.encrypt(
-    JSON.stringify(requestData),
-    ENCRYPTION_KEY || assertKey(SECRET_KEY, () => errorThrower.throwMissingSecretKeyError()),
-  ).toString();
+  const maybeAccountlessKey =
+    process.env.NODE_ENV === 'production'
+      ? ENCRYPTION_KEY || assertKey(SECRET_KEY, () => errorThrower.throwMissingSecretKeyError())
+      : ENCRYPTION_KEY || SECRET_KEY || ACCOUNTLESS_ENCRYPTION_KEY;
```

Comment:
If we think the predefined key is extremely unsafe, we can use the Accountless Secret key that it's stored in the cookie.